pkg/catalog/util.go

```diff
@@ -125,6 +125,24 @@ func Attribute(a Attr, query string) string {
 	return ""
 }
 
+func ServiceMetadata(a Attr) string {
+	metadata := strings.Builder{}
+	queries := []string{
+		"Repo.URL",
```

Comment:
Sorry, oversight on my part, can we make these lower-case to match other attributes etc. that exist such as "@package"?